cool-core/src/test/java/com/nus/cool/core/refactor/CohortStoreTest.java

```diff
@@ -0,0 +1,52 @@
+package com.nus.cool.core.refactor;
+
+import com.google.common.io.Files;
+import com.nus.cool.core.cohort.refactor.storage.CohortRSStr;
+import com.nus.cool.core.cohort.refactor.storage.CohortWSStr;
+import java.io.DataOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.ByteOrder;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import org.testng.annotations.Test;
+
+/**
+ * Testing CohortStoreTest.
+ */
+public class CohortStoreTest {
+
+  /**
+   *
+   * Testing cohort query.
+   */
+  @Test()
+  public void storeCohort() throws IOException {
+
+    System.out.println(System.getProperty("user.dir"));
+
+    ArrayList<String> testData =
+        new ArrayList<>(Arrays.asList("Userid1", "Userid2", "Userid3", "Userid4", "Userid5"));
+
+    CohortWSStr cws = new CohortWSStr(StandardCharsets.UTF_8);
+
+    for (String userID : testData) {
+      cws.addCubletResults(userID);
+    }
+
+    // output file
+    String fileName = "TestCohort";
+    File cubemeta = new File(System.getProperty("user.dir"), fileName);
```

Comment:
Please format this unit test (i.e., add the DataProvider structure in this unit test).

Reply:
This does not rely on any existing dataset locally. It's a unit test for just a read-write class.  In this case, we use fake data, such as user1 to use5; store them locally; read them back; compare results; delete the file just written to disk.

Reply:
test data is the sample you want to use. It is better to generate it in the data provider function. Thanks

Reply:
Updated